engine/runtime/src/main/java/org/enso/interpreter/instrument/ReplDebuggerInstrument.java

```diff
@@ -88,6 +89,7 @@ protected OptionDescriptors getOptionDescriptors() {
   public static class ReplExecutionEventNode extends ExecutionEventNode {
     private @Child EvalNode evalNode = EvalNode.buildWithResultScopeCapture();
     private @Child ToJavaStringNode toJavaStringNode = ToJavaStringNode.build();
+    private @Child AnyToTextNode toTextNode = AnyToTextNode.build();
```

Comment:
you're not using this anymore

Reply:
Thanks for the catch :fishing_pole_and_fish: